.gitignore

```diff
@@ -0,0 +1 @@
+src/build
```

Comment:
Looks like you add the gitignore on both branches – nothing wrong with that just be careful to not introduce a merge conflict

Reply:
got it